src/amo/components/SearchForm/Suggestion.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+/* eslint react/no-array-index-key: 0 */
+import React from 'react';
+import escapeStringRegexp from 'escape-string-regexp';
+
+type Props = {|
+  search: string,
```

Comment:
I think `searchQuery` or `query` might work better here because it's the term we use across the rest of the app for a string used to search the API. This is part of why I was confused by the code in `Suggestion`